adam/visualization/make_scenes.py

```diff
@@ -97,20 +89,36 @@ def main(params: Parameters) -> None:
             render_obj, viz, property_map
         )  # bind visualizer and properties to nested obj rendering function
         bound_render_nested_obj = partial(render_obj_nested, viz, property_map)
-        # render each object in graph
-        SceneCreator.graph_for_each_top_level(
-            obj_graph, bound_render_obj, bound_render_nested_obj
-        )
+        try:
+            # render each object in graph
+            SceneCreator.graph_for_each_top_level(
+                obj_graph, bound_render_obj, bound_render_nested_obj
+            )
+        except RuntimeError:
```

Comment:
There's an error thrown when the system tries to render something Irregular - might be better as a `NotImplementedError` or just not caught at all.

Reply:
Can you give an example of rendering something irregular?

Reply:
Unfortunately there are several examples (none of which are currently handled): liquids are irregular (but I do have a puddle shaped model lying around that could be used for them), things like car bodies are irregular, and the backs of chairs are irregular (curved like an office chair presumably).

Reply:
This is 'irregular' in terms of Geon type

Reply:
Got it. Can you add this information in a comment there?

Reply:
Actually, it might be better to remove the catch and have an `omit_irregular=True` argument to `graph_for_each_top_level`. As it stands catching the exception could sometimes hide problems and suppress useful debugging information.

Reply:
I could not find a good place to fit that sort of switch, but I'll make an issue to address it + the general clunkiness of swapping out which curriculum is fed into the visualizer.

Reply:
In the meantime, can you narrow this from `RuntimeError` to `NotImplementedError`